### PR TITLE
Added "Lorem Ipsum Generator"

### PR DESCRIPTION
**Pull request by Tomkov1c**

### CODE REVIEW

**Comment by jjw24** [comment]
Hi there, it looks like you haven't published a GitHub release. This is usually done via a GitHub workflow file. Feel free to have a look at other python plugins.

Once done please update the download url to the zip file you released on GitHub.

**Comment by Tomkov1c** [comment]
> Hi there, it looks like you haven't published a GitHub release. This is usually done via a GitHub workflow file. Feel free to have a look at other python plugins.
> 
> Once done please update the download url to the zip file you released on GitHub.

Done

**Comment by jjw24** [comment]
Hi there, sorry the workflow is not working quite right. Follow thisnrepo as an example:

https://github.com/z1nc0r3/Unicode-Finder-Plugin/blob/main/.github/workflows/Publish%20Release.yml

https://github.com/z1nc0r3/Unicode-Finder-Plugin/releases

**Comment by Tomkov1c** [comment]
> Hi there, sorry the workflow is not working quite right. Follow this repo as an example:
> 
> https://github.com/z1nc0r3/Unicode-Finder-Plugin/blob/main/.github/workflows/Publish%20Release.yml
> 
> https://github.com/z1nc0r3/Unicode-Finder-Plugin/releases

Done

File: plugins.json [hunk 1]
```diff
@@ -2317,6 +2317,19 @@
...
+    },
+    {
+      "ID": "e483e4f2-ec67-3c54-2aea-f823c5e5f61b",  
+      "Name": "Lorem Ipsum Generator",
+      "Description": "Generates Lorem Ipsum and copies it to your clipboard.",
+      "Author": "Tom Kliner",
+      "Version": "1.0.0",
+      "Language": "python",
+      "Website": "https://github.com/Tomkov1c/Lorem-Ipsum-Generator",
+      "UrlDownload":"https://github.com/Tomkov1c/Lorem-Ipsum-Generator/releases/tag/Releaser",
```
Comment (jjw24) [inline plugins.json:2330]:
```suggestion
      "UrlDownload":"https://github.com/Tomkov1c/Lorem-Ipsum-Generator/releases/download/v1.0.0/Flow.Launcher.Plugin.LoremIpsumGenerator.zip",
```

File: plugins.json [hunk 2]
```diff
@@ -2317,6 +2317,19 @@
...
+    {
+      "ID": "e483e4f2-ec67-3c54-2aea-f823c5e5f61b",  
+      "Name": "Lorem Ipsum Generator",
+      "Description": "Generates Lorem Ipsum and copies it to your clipboard.",
+      "Author": "Tom Kliner",
+      "Version": "1.0.0",
+      "Language": "python",
+      "Website": "https://github.com/Tomkov1c/Lorem-Ipsum-Generator",
+      "UrlDownload":"https://github.com/Tomkov1c/Lorem-Ipsum-Generator/releases/download/v1.0.0/Flow.Launcher.Plugin.LoremIpsumGenerator.zip",
+      "UrlSourceCode": "https://github.com/Tomkov1c/Lorem-Ipsum-Generator/releases/tag/Release",
```
Comment (jjw24) [inline plugins.json:2331]:
```suggestion
      "UrlSourceCode": "https://github.com/Tomkov1c/Lorem-Ipsum-Generator",
```

File: plugins.json [hunk 3]
```diff
@@ -2317,6 +2317,19 @@
...
+      "ID": "e483e4f2-ec67-3c54-2aea-f823c5e5f61b",  
+      "Name": "Lorem Ipsum Generator",
+      "Description": "Generates Lorem Ipsum and copies it to your clipboard.",
+      "Author": "Tom Kliner",
+      "Version": "1.0.0",
+      "Language": "python",
+      "Website": "https://github.com/Tomkov1c/Lorem-Ipsum-Generator",
+      "UrlDownload":"https://github.com/Tomkov1c/Lorem-Ipsum-Generator/releases/download/v1.0.0/Flow.Launcher.Plugin.LoremIpsumGenerator.zip",
+      "UrlSourceCode": "https://github.com/Tomkov1c/Lorem-Ipsum-Generator",
+      "IcoPath": "https://github.com/Tomkov1c/Lorem-Ipsum-Generator/blob/main/Images/app.png"
```
Comment (jjw24) [inline plugins.json:2332]:
```suggestion
      "IcoPath": "https://cdn.jsdelivr.net/gh/Tomkov1c/Lorem-Ipsum-Generator@main/Images/app.png"
```

File: plugins.json [hunk 4]
```diff
@@ -2317,6 +2317,19 @@
...
+      "Name": "Lorem Ipsum Generator",
+      "Description": "Generates Lorem Ipsum and copies it to your clipboard.",
+      "Author": "Tom Kliner",
+      "Version": "1.0.0",
+      "Language": "python",
+      "Website": "https://github.com/Tomkov1c/Lorem-Ipsum-Generator",
+      "UrlDownload":"https://github.com/Tomkov1c/Lorem-Ipsum-Generator/releases/download/v1.0.0/Flow.Launcher.Plugin.LoremIpsumGenerator.zip",
+      "UrlSourceCode": "https://github.com/Tomkov1c/Lorem-Ipsum-Generator",
+      "IcoPath": "https://cdn.jsdelivr.net/gh/Tomkov1c/Lorem-Ipsum-Generator@main/Images/app.png"
+      "LatestReleaseDate": "2024-05-24T03:13:31Z"
```
Comment (jjw24) [inline plugins.json:2333]:
```suggestion
```

**Pull request merged**